.github/workflows/tests.yml

```diff
@@ -122,7 +122,7 @@ jobs:
   process-results:
     needs: tests
     name: Combine separate benchmark results
-    if: always() && github.repository == 'coiled/coiled-runtime'
+    if: always() && github.repository == 'coiled/coiled-runtime' && needs.tests.result != 'cancelled'
```

Comment:
@ian-r-rose I agree that if tests were canceled we don't care about regressions. I added this, I assume this will mean that regression will be skipped, and the change on line 224 can be removed right?
 

In that case I could remove

Reply:
I think that's not right, since the regressions condition has an `always()`. I meant that we can add this to the regressions step, rather than the process-results step